packages/wallets/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -27,14 +27,10 @@ const JurisdictionModal = () => {
     const marketType = getModalState('marketType') ?? 'all';
     const platform = getModalState('platform') ?? PlatformDetails.mt5.platform;
 
-    const { title } = MarketTypeDetails[marketType];
-
     const toggleDynamicLeverage = useCallback(() => {
         setIsDynamicLeverageVisible(!isDynamicLeverageVisible);
     }, [isDynamicLeverageVisible, setIsDynamicLeverageVisible]);
 
-    const jurisdictionTitle = `Choose a jurisdiction for your ${PlatformDetails.mt5.title} ${title} account`;
```

Comment:
You're removing the details of the modal heading based on the new design, its design needs to be discussed. Let's stick to the production behavior until we confirm that we need to remove this. There's a high chance that they missed this part.

Reply:
I'll get akie's confirmation on this